src/commands/run.js

```diff
@@ -132,6 +135,7 @@ exports.handler = function ({
     appInitArgs,
     clientVersion,
     clientPort
```

Comment:
Missing a `,` here

Reply:
Good catch! 👍 

Slipped through during rebase.

Reply:
This is now fixed.